src/Saxulum/Console/Provider/ConsoleProvider.php

```diff
@@ -32,6 +33,11 @@ public function register(Container $container)
 
         $container['console'] = function () use ($container) {
             $console = new ConsoleApplication($container);
+            
+            if (!is_null($container['console.dispatcher'])) {
```

Comment:
I have chosen this implementation so that the user can *always* override the ``console.dispatcher`` setting, e.g. when the user decides to use another dispatcher instead of the one provided by Silex. ;)